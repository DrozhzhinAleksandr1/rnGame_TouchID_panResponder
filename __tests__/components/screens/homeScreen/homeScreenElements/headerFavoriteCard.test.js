import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { localization } from '../../../../../app/common/localization';
import { HeaderFavoriteCard } from '../../../../../app/components/screens/homeScreen/homeScreenElement/headerFavoriteCard';

const mockItem = {
    Ask: "0.92835",
    Bid: "0.92802",
    Diff: "0.00010",
    ESV: "0.92802",
    FullName: "AUD/CAD",
    ID: 1,
    Market: "Currency",
    Max: "0.92933",
    Min: "0.92696",
    Name: "AUDCAD",
    Per: 1539938640,
    Rate: "0.0108",
    SSV: "0.92792",
    Sess: "Open",
    Timestamp: 1539938689948,
}

it('create snapshot HeaderFavoriteCard', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<HeaderFavoriteCard item={mockItem} CurrentLanguage={localization.ru} isInfoOpened={true} isGraphicOpened={true} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
