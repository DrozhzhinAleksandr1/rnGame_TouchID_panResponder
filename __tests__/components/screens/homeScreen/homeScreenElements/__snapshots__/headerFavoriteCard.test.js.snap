// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot HeaderFavoriteCard 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#1D2E3E",
      "shadowColor": "rgba(0, 0, 0, 0.15)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 4,
      "zIndex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "backgroundColor": "#1D2E3E",
        "flexDirection": "row",
        "height": 48,
        "justifyContent": "space-between",
      }
    }
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "height": "100%",
          "justifyContent": "center",
          "width": "25%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Roboto-Regular",
            "fontSize": 16,
            "lineHeight": 19,
            "marginTop": 5,
          }
        }
      >
        AUD/CAD
      </Text>
    </Component>
    <Component
      style={
        Object {
          "flexDirection": "row",
          "height": "100%",
          "justifyContent": "space-between",
          "width": "75%",
        }
      }
    >
      <OpenCardQuotation
        numbFromServer="0.92835"
      />
      <OpenCardQuotation
        numbFromServer="0.92802"
      />
      <Component
        style={
          Object {
            "backgroundColor": "rgba(23, 37, 51, 0.86)",
            "height": 28,
            "left": 253.75,
            "position": "absolute",
            "top": 10,
            "width": 40,
            "zIndex": 9999,
          }
        }
      >
        <Component
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#fff",
                "fontFamily": "Roboto-Regular",
                "fontSize": 14,
              }
            }
          >
            3.3
          </Text>
        </Component>
      </Component>
    </Component>
  </Component>
  <OpenCell
    CurrentLanguage={
      Object {
        "A letter to confirm your data...": "Письмо для подтверждения Ваших данных и завершения процесса регистрации отправлено на указаный e-mail",
        "Already have an account?": "Уже есть аккаунт ?",
        "AnotherProblem": "Обнаружена непредвиденная ошибка",
        "Are you sure you want to log out?": "Уверены, что хотите выйти ?",
        "Buy": "Покупать",
        "Cancel": "Отмена",
        "ConnectionError": "Не удалось войти из-за проблем с соединением. Проверьте подключение к Интернету",
        "Country": "Страна",
        "Email-entry error": "Ошибка ввода e-mail",
        "EmailIsExist": "К сожалению, пользователь с данным email уже зарегистрирован",
        "EmailOrPassWrong": "Некорректный адрес электронной почты или пароль",
        "EmailWrong": "Некорректный адрес электронной почты",
        "Enter your country": "Укажите страну",
        "Forgot password?": "Забыли пароль?",
        "Hour": "час",
        "I have read and agree with": "Заполняя форму регистрации,",
        "Indicators": "Показатели",
        "Instructions for password recovery...": "Инструкция

 по восстановлению пароля отправлена на почту пользователя",
        "Invalid email address or password": "Неверный адрес электронной почты или пароль",
        "Invalid email address or server error": "Неверный адрес электронной почты или ошибка сервера",
        "InvalidParameters": "Параметры, введенные во время регистрации, некорректны",
        "LastUpdate": "Последнее обновление",
        "MovingAverages": "Скользящие средние",
        "Neutral": "Нейтрально",
        "No": "Нет",
        "OK": "ОК",
        "Password failed": "Ошибка ввода пароля",
        "Password recovery": "Восстановление пароля",
        "PasswordRecoverySuccess": "Инструкция по восстановлению пароля отправлена на почту пользователя",
        "Please specify the email you used when registering": "Пожалуйста, укажите e-mail, который Вы использовали при регистрации",
        "Quote": "Котировка",
        "RegSuccess": "Регистрация прошла успешно",
        "Registration": "Регистрация",
        "Sell": "Продавать",
        "ServerUnreachable": "Ошибка при подключении к серверу",
        "Something went wrong": "что - то пошло не так",
        "Thank you for joining us!": "Спасибо,

 что присоеденились к нам!",
        "To get full access to the instruments...": "Чтобы получить полный доступ к инструментам, Вам необходимо войти в систему",
        "Yes": "Да",
        "cellItem": Object {
          "opencellItem": Object {
            "closeTitle": "Закрытые:",
            "diferenceTitle": "Разница:",
            "maxTitle": "Макс.:",
            "minTitle": "Мин.:",
            "openTitle": "Открытые:",
            "percentdifereceTitle": "Разница%:",
          },
          "titleOfBuy": "Покупка",
          "titleOfSell": "Продажа",
        },
        "confirmPasswordHints": Object {
          "emptyName": "Поле обязательно для заполнения",
          "emptyNameSO": "Поле обязательно для заполнения",
          "length": "Пароль должен содержать не менее 5 символов",
          "lengthSO": "Пароль должен содержать не менее 5 символов",
          "minOneDigit": "В пароле должны присутствовать буквы и цифры",
          "minOneDigitSO": "В пароле должны присутствовать буквы и цифры",
          "minOneLeter": "В пароле должны присутствовать буквы и цифры",
          "minOneLeterSO": "В пароле должны присутствовать буквы и цифры",
          "onlyLatinAndDigits": "5-20 знаков, английские буквы и цифры",
          "onlyLatinAndDigitsSO": "5-20 знаков, английские буквы и цифры",
        },
        "confirmPasswordMatch": "Пароли должны совпадать",
        "countries": Object {
          "London": "Лондон",
          "Moscow": "Москва",
          "New York": "Нью-Йорк",
          "Sydney": "Сидней",
          "Tokyo": "Токио",
        },
        "emailExists": "Такой адрес уже существует",
        "emailHints": Object {
          "emptyName": "Поле обязательно для заполнения",
          "emptyNameSO": "Поле обязательно для заполнения",
          "incorrect": "Введите корректный формат эл. почты",
          "incorrectSO": "Введите корректный формат эл. почты",
          "symbols": "Допустимы символы \\".\\", \\"_\\", \\"-\\", \\"+\\", \\"@\\"",
          "symbolsSO": "Допустимы символы \\".\\", \\"_\\", \\"-\\", \\"@\\"",
        },
        "enter": "Вход",
        "error!": "Ошибка!",
        "errorMessage": Object {
          "email": "Неверный email",
          "lastName": "Неверная фамилия",
          "name": "Неверное имя",
          "phoneNumber": "Неверно указан номер телефона",
        },
        "errorTypes": Object {
          "3": "Слишком простой пароль",
          "6": "Пользователь с этим e-mail уже зарегистрирован.",
        },
        "errors": Object {
          "errorEmail": "Ошибка ввода e-mail",
          "errorFirstName": "Ошибка ввода имени",
          "errorLastName": "Ошибка ввода фамилии",
        },
        "exit": "Выход",
        "introduced": "Ознакомлен",
        "lastNameHints": Object {
          "emptyName": "Поле обязательно для заполнения",
          "emptyNameSO": "Поле обязательно для заполнения",
          "incorrect": "Введите корректные данные",
          "incorrectSO": "Введите корректные данные",
        },
        "login": "Войти",
        "min": "мин",
        "nameHints": Object {
          "emptyName": "Поле обязательно для заполнения",
          "emptyNameSO": "Поле обязательно для заполнения",
          "incorrect": "Введите корректные данные",
          "incorrectSO": "Введите корректные данные",
        },
        "notifications": "Уведомления",
        "passwordHints": Object {
          "emptyName": "Поле обязательно для заполнения",
          "emptyNameSO": "Поле обязательно для заполнения",
          "length": "Пароль должен содержать не менее 5 символов",
          "lengthSO": "Пароль должен содержать не менее 5 символов",
          "minOneDigit": "В пароле должны присутствовать буквы и цифры",
          "minOneDigitSO": "В пароле должны присутствовать буквы и цифры",
          "minOneLeter": "В пароле должны присутствовать буквы и цифры",
          "minOneLeterSO": "В пароле должны присутствовать буквы и цифры",
          "onlyLatinAndDigits": "5-20 знаков, английские буквы и цифры",
          "onlyLatinAndDigitsSO": "5-20 знаков, английские буквы и цифры",
        },
        "phoneNumberHints": Object {
          "emptyName": "Поле обязательно для заполнения",
          "emptyNameSO": "Поле обязательно для заполнения",
          "incorrect": "Введите правильный номер телефона",
          "incorrectSO": "Введите правильный номер телефона",
          "onlyNumbers": "Введите только цифры",
          "onlyNumbersSO": "Введите только цифры",
        },
        "placeHolder": Object {
          "confirmPassword": "Подтверждение пароля",
          "email": "Email",
          "lastName": "Фамилия",
          "name": "Имя",
          "password": "Пароль",
          "phoneNumber": "Телефон",
        },
        "sendBtnText": "Отправить",
        "signup": "Зарегистрируйтесь",
        "signup2": "Зарегистрироваться",
        "statusCodes": Object {
          "500": "Регистрация недоступна,повторите попытку.",
        },
        "stillDontHaveAccaunt": "Ещё нет учётной записи?",
        "tabbar": Object {
          "currency": "Валюты",
          "goods": "Товары",
          "indexes": "Индексы",
          "shares": "Акции",
        },
        "turnBtn": "Свернуть все",
        "user agreement": "я принимаю условия обработки персональных данных.",
        "user agreement2": "Пользовательское соглашение",
      }
    }
    Diff="0.00010"
    ESV="0.92802"
    Max="0.92933"
    Min="0.92696"
    Rate="0.0108"
    SSV="0.92792"
    isGraphicOpened={true}
    isInfoOpened={true}
  />
</Component>
`;
